src/Senders/Jaeger.Senders.Grpc/GrpcSender.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Grpc.Core;
+using Jaeger.ApiV2;
+using Jaeger.Exceptions;
+using Jaeger.Senders.Grpc.Protocols;
+using GrpcProcess = Jaeger.ApiV2.Process;
+using GrpcSpan = Jaeger.ApiV2.Span;
+
+namespace Jaeger.Senders.Grpc
+{
+    /// <summary>
+    /// GrpcSender provides an implementation to transport spans over HTTP using GRPC.
+    /// </summary>
+    public class GrpcSender : ISender
+    {
+        public const string DefaultCollectorGrpcTarget = "localhost:14250";
+        public const int MaxPacketSize = 65000;
```

Comment:
There's no expectation that gRPC messages are limited to 65000 bytes, this number only applies to UDP servers in the jaeger-agent. I am not sure this is sensible default, it's worth benchmarking. It could also be highly dependent on the network conditions, so probably worth exposing to the user via configuration.

Reply:
This has to be 4 MB (or a little less depending on process size) as that‘s the default for gRPC if not defined manually.

Reply:
Updated to 4 MB. Resolve if OK for you.

Reply:
4MB sounds like a lot of data to buffer. Is there a timer-based flush in addition to size limit?

Reply:
Timer based flush is in RemoteReporter for all senders.